atom/common/crash_reporter/crash_reporter_win.cc

```diff
@@ -54,6 +54,7 @@ void CrashReporterWin::InitBreakpad(const std::string& product_name,
   if (waiting_event != INVALID_HANDLE_VALUE)
     WaitForSingleObject(waiting_event, 1000);
 
+  breakpad_.reset(nullptr);
```

Comment:
`scoped_ptr::reset` releases the old value before [assignment](https://code.google.com/p/chromium/codesearch#chromium/src/base/memory/scoped_ptr.h&sq=package:chromium&rcl=1431776765&l=226).